libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1474,9 +1503,15 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 		// but all parameters of ecrecover are value types anyway.
 		encodeInPlace = false;
 	bool encodeForLibraryCall = funKind == FunctionType::Kind::DelegateCall;
-	solUnimplementedAssert(!encodeInPlace, "");
-	solUnimplementedAssert(funType.padArguments(), "");
```

Comment:
Isn't it worth keeping this assert for the `!encodeInPlace` case?

Reply:
You mean like that: https://github.com/ethereum/solidity/pull/8763/files#diff-9e73c6daf061b8b0e6dff4531c75e83eR1539